<launch>
  <!-- Launchfile Arguments (optional)-->
  <arg name="robot_name" default="wam"/>

  <!-- Robot Model & Kinematics-->
  <!--param name="robot_description" command="$(find xacro)/xacro.py '$(find barrett_model)/robots/wam_7dof_$(arg robot_name).urdf.xacro' robot_name:=$(arg robot_name)"/-->
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find barrett_model)/robots/wam_7dof_$(arg robot_name).urdf.xacro'"/>
  <param name="root_link" value="$(arg robot_name)/FixedLink"/>
  <param name="tip_link" value="$(arg robot_name)/LowerWristYawLink"/>

  <!-- Component Configuration -->
  <rosparam param="initial_positions">
    [0.0, rad(-pi/2), 0.0, rad(pi), 0.0, 0.0,  0.0]
  </rosparam>
</launch>
